ucentral.state.pretty.json

```diff
@@ -75,6 +75,13 @@
                     }
                 }
             }
+        },
```

Comment:
re note for the ref in state/schema.yml, this is the result of the unrealized reference